assets/src/edit-story/components/library/panes/pageTemplates/pageTemplates.js

```diff
@@ -212,6 +216,21 @@ function PageTemplates({
           />
         </TemplatesToggle>
       </ActionRow>
+      <FirstRow pageSize={pageSize}>
```

Comment:
@barklund I'm thinking of how to use the virtualized list while having the Save new template button as the first item instead of the first Page.

One option could be this as in this PR: moving the first row out of the virtualized, containing the new button and the first page template. The downside of this is that the first page template would need extra handling in terms of keyboard handling and would not belong to the list anymore at all.

An alternative could be adding the New template button to the virtualized list as well. Downsized of this -- it's not really a page at all and might need some "hackiness" for adding to the list. Might still be less hacky than what the current approach needs to make the first page part of the list again in terms of accessibility.

So without having tried it out it seems that perhaps it's better to force the non-page button to be part of the list instead.

Thoughts?

Reply:
Creating a separate issue for this part.